cassiopeia/data/utilities.py

```diff
@@ -36,7 +45,12 @@ def get_lca_characters(
         assert len(i) == k
     lca_vec = [0] * len(vecs[0])
     for i in range(k):
-        chars = set([vec[i] for vec in vecs])
+        chars = set()
+        for vec in vecs:
+            if isinstance(vec[i], tuple):
```

Comment:
why not use `is_ambiguous_state`, since you defined it at the bottom?

Reply:
Yep, good catch